src/sql/src/session/vars/definitions.rs

```diff
@@ -1459,6 +1459,13 @@ pub static ENABLE_COMPUTE_CHUNKED_STACK: VarDefinition = VarDefinition::new(
     true,
 );
 
+pub static ENABLE_LGALLOC_EAGER_RECLAMATION: VarDefinition = VarDefinition::new(
+    "enable_lgalloc_eager_reclamation",
+    value!(bool; true),
+    "Enable lgalloc's eager return behavior.",
+    true,
+);
```

Comment:
Why can't you define this via `feature_flags!`? That would a bit of boilerplate I think.

Reply:
Oh, because the implementation predates the macro! Happy to change it, but I'd prefer in a follow-up PR if that's alright with you!